editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/utils/CamelUtils.java

```diff
@@ -80,4 +80,9 @@ public static IProject project() {
         CamelDesignEditor editor = CamelUtils.getDiagramEditor();
         return editor == null ? null : editor.getWorkspaceProject();
 	}
+	
+	public static IProject project(IFeatureProvider fp) {
```

Comment:
Indention of that class looks odd here on web.

Reply:
effectively some spaces were used